TaylorSource/CoreData/NSFRCDatasource.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+import CoreData
+
+public struct NSFRCDatasource<
+    Factory
+    where
+    Factory: _FactoryType,
+    Factory.ViewType: NSFRCIndexedUpdateConsumer,
+    Factory.CellIndexType == NSFRCCellIndex,
+Factory.SupplementaryIndexType == NSFRCSupplementaryIndex>: DatasourceType {
+    
+    public typealias FactoryType = Factory
+    
+    public let identifier: String
+    public let factory: Factory
+    public var title: String? = .None
+    
+    public let updateHandler = NSFRCUpdateHandler()
+    public var selectionManager = IndexPathSelectionManager()
+    
+    public init(id: String, fetchedResultsController: NSFetchedResultsController, factory f: Factory) {
+        fetchedResultsController.delegate = self.updateHandler
+        self.fetchedResultsController = fetchedResultsController
```

Comment:
Couple of points...

I tend to put all properties together at the top of a type. I don't both with using `// MARK: - Private` in the middle of the type, because the code should already have `public` or `private` access controls.

Because `init` then probably comes directly after the properties, it's then easy to read. I'm putting this comment here, because I looked at this, and thought... Where is the `fetchedResultsController` property? I don't really get the logic of why private properties get put right at the bottom of the type? 🤔